css/_uswds-theme-custom-styles.scss

```diff
@@ -519,6 +519,19 @@ img {
   }
 }
 
+.site-figure {
+  @include u-display("flex");
+  @include u-flex("column");
+  @include u-flex("align-center");
+  @include u-margin(0);
+}
+
+.site-figcaption {
+  @include u-font-size("lang", "2xs");
+  @include u-margin-y(2);
+  color: color("gray-60");
+}
+
```

Comment:
@mejiaj These might be my first styles created for site. Can you review these updates and let me know if they follow our standards?

Reply:
Sure, added my review below. (https://github.com/uswds/uswds-site/pull/1751#pullrequestreview-1095073705)